docs/production/deployment.md

```diff
@@ -661,15 +661,6 @@ SSL/TLS termination.
 Set to the port number if you [prefer to listen on a port other than
 443](#using-an-alternate-port).
 
-#### `no_serve_uploads`
-
-To enable the [the S3 uploads backend][s3-uploads], one needs to both
-configure `settings.py` and set this to true to configure
-`nginx`. Remove this field to return to the local uploads backend (any
-non-empty value is currently equivalent to true).
-
-[s3-uploads]: upload-backends.md#s3-backend-configuration
```

Comment:
We probably needs something in the Changelog "upgrade notes" related to this adjustment.

Reply:
Do you want me to add a Changelog entry as part of this series?

Reply:
An immediate follow-up is fine, but because folks can upgrade to `main`, we generally aim to have "Upgrade notes" be added as we merge things that deserve them.

Reply:
- #24030